BrowserKit/Sources/Shared/Cancellable.swift

```diff
@@ -1,55 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Foundation
-
-// TODO: FXIOS-13184 Remove deferred code or validate it is sendable
-// Also validate the T type is Sendable (actually protected) in all methods.
-open class CancellableDeferred<T: Sendable>: Deferred<T>, @unchecked Sendable {
```

Comment:
This is used under `SwiftData` [here](https://github.com/mozilla-mobile/firefox-ios/blob/c7d1b81b124a09951517466a18b6278d91dbd8a1/firefox-ios/Storage/ThirdParty/SwiftData.swift#L48), please revert as this code is still used under Firefox iOS.